tracer/src/Datadog.Trace/SpanContext.cs

```diff
@@ -110,6 +117,16 @@ private SpanContext(ulong? traceId, string serviceName)
         /// </summary>
         internal string Origin { get; set; }
 
+        /// <summary>
+        /// Gets or sets a collection of propagated internal Datadog tags,
+        /// formatted as "key1=value1,key2=value2".
```

Comment:
Unrelated really as we're doing pass-through anyway, but we've run into issues where we separate tags with a comma instead of a space (i.e. we split on "`,`", agent splits on space "` `"). The work-around is to recommend separating by both, i.e. "`, `". Maybe we should recommend that here too for consistency?
```suggestion
        /// formatted as "key1=value1, key2=value2".
```

Reply:
I think that's a result of not having well-defined RFCs in the past. However, this header has well-defined syntax in eBNF format and I don't want to deviate from the RFC in this code comment.

```
tagset = ( tag, { ",", tag } ) | "";
tag = ( identifier - space ), "=", identifier;
identifier = allowed characters, { allowed characters };
allowed characters = ( ? ASCII characters 32-126 ? - equal or comma );
equal or comma = "=" | ",";
space = " ";
```